src/pages/customize.tsx

```diff
@@ -111,22 +112,22 @@ export default function Customize(props: CustomizePageProps) {
                       config: itemConfiguration,
                     });
                     router.push("/review");
-                  }} 
-                  href="/create/ETH">
```

Comment:
not sure how this got here :/ but it was breaking the customize -> cart flow. long term we should add some tests to avoid things like this

Reply:
There are some really simple yet highly effective tests we can do for this. user flow mocking would be best use case for us. basically a program will click through the site and make sure expected outcomes return as expected

one I'm familiar with is: https://github.com/getgauge/taiko